### DISCUSSION TITLE
Online games

### DISCUSSION THREAD

**Discussion by blackmind30** [termux/termux-app#2469] [Q&A] [answered]
How to use kisskiss-spin the bottle game in termux

**Comment by khansaad1275** [comment]
> How to use kisskiss-spin the bottle game in termux

Please share more information about the game.

**Comment by agnostic-apollo** [comment, marked as answer]
These are roughly the games provided by termux. You can't run android app games in the termux terminal. Some additional linux distro supported games should be possible inside graphical env.

https://github.com/termux/game-packages/tree/master/packages

https://wiki.termux.com/wiki/Graphical_Environment
